src/main/java/com/amazonaws/kinesisvideo/utils/Constants.java

```diff
@@ -0,0 +1,12 @@
+package com.amazonaws.kinesisvideo.utils;
+
+public class Constants {
+    /**
+     * SDK identifier
+     */
+    public static final String APP_NAME = "aws-kvs-webrtc-android-client";
+    /**
+     * SDK version identifier
+     */
+    public static final String VERSION = "1.0.0";
+}
```

Comment:
Nice :)

Perhaps these can be added here too:

`X-Amz-ChannelARN`, `X-Amz-ClientId`

Reply:
Done